packages/app-builder-lib/src/codeSign/windowsCodeSign.ts

```diff
@@ -319,3 +319,10 @@ async function getToolPath(isWin = process.platform === "win32"): Promise<ToolIn
     return { path: path.join(vendorPath, process.platform, "osslsigncode") }
   }
 }
+
+async function getPSCmd(vm: VmManager): Promise<string> {
+  return await vm
+    .exec("powershell.exe", ["-NoProfile", "-NonInteractive", "-Command", `Get-Command pwsh.exe`])
+    .then(() => "pwsh.exe")
+    .catch(() => "powershell.exe")
+}
```

Comment:
I think it may be worthwhile to add some debug logging here to help any debugging in the future. What are your thoughts?
```
vm
.exec("powershell.exe", ["-NoProfile", "-NonInteractive", "-Command", `Get-Command pwsh.exe`])
.then(() => {
    log.debug(null, "identified pwsh.exe for executing code signing")
    return "pwsh.exe"
})
.catch(() => {
    log.debug(null, "unable to find pwsh.exe, falling back to powershell.exe")
    return "powershell.exe"
})
```

Reply:
Good idea! Done.